src/routers/TabRouter.js

```diff
@@ -32,6 +32,8 @@ export default (
   const initialRouteIndex = order.indexOf(initialRouteName);
   const backBehavior = config.backBehavior || 'initialRoute';
   const shouldBackNavigateToInitialRoute = backBehavior === 'initialRoute';
+  const shouldBackNavigateToLastActiveTab = backBehavior === 'previousRoute';
+  const backstack = [];
```

Comment:
@ericvicenti @satya164 or can someone please confirm what is required, so I don't go the wrong way on this. I am new to this library. Thank you.